Ocean-back/Controllers/QuizQuestionController.cs

```diff
@@ -11,11 +11,11 @@ public class QuizQuestionController : ControllerBase
         private readonly ISender _sender;
         public QuizQuestionController(ISender sender) => _sender = sender;
 
-        //[HttpGet("all")]
-        //public async Task<IActionResult> GetQuestions() => Ok(await _sender.Send(new GetQuestions.Request()));
+        [HttpGet("all")]
+        public async Task<IActionResult> GetQuestions() => Ok(await _sender.Send(new GetQuestions.Request()));
 
-        //[HttpGet("{questionId}")]
-        //public async Task<IActionResult> GetQuestion(Guid charId) => Ok(await _sender.Send(new GetQuestion.Request { QuestionId = questionId }));
+        [HttpGet("{questionId}")]
+        public async Task<IActionResult> GetQuestion(Guid questionId) => Ok(await _sender.Send(new GetQuestion.Request { CharId = questionId }));
 
```

Comment:
QuestionId instead of CharId. ChardId is for Characteristic, this is QuizQuestion

Reply:
![kjamil_response](https://github.com/WebZaLeb/OCEAN-WebApp/assets/46653449/42ddf362-e2bd-42ea-be13-e7181299eb51)